types/types.go

```diff
@@ -252,6 +252,8 @@ type Info struct {
 	ClusterStore       string
 	ClusterAdvertise   string
 	SecurityOptions    []string
+	Runtimes           map[string]Runtime
```

Comment:
Sorry my comment was ambiguous: I meant I'm not sure we should expose the path. But I guess it shouldn't hurt, let's do it :+1: